packages/jest-environment-enzyme/README.md

```diff
@@ -17,11 +17,18 @@ But npm works too!
 $ npm install jest-environment-enzyme --save-dev
 ```
 
-### Setup
+With this library, you don't need to have your app install `enzyme`, but you have to install one of the `enzyme-adapter`.
+
+```
+yarn add enzyme-adapter-react-* --dev
+```
 
-With this library, you don't need to have your app install enzyme, or an enzyme-adapter. We'll take care of it for you!
+Where `*` is your app's React version.
```

Comment:
Change this to to 

> enzyme-adapter-* --dev
> 
> Where '*" is your app's adapter.

Reason being is that the adapters were made to support other frameworks like inferno or preact. There will be adapters for those and this works for them.